### PR TITLE
*: more accurate time detail in kv response

### LINKED ISSUE #14262: suspended-time should be passed through to TiDB to avoid confusion
## Bug Report
in PR #9257, we introduced the suspended time in the tracker. However this information is included in process time in TiDB side, which misleads the investigation. As slow process time typically means low IO or CPU resources. 
It should be exposed as suspended time itself in TiDB. 

### What version of TiKV are you using?

6.5 Or newer

### Steps to reproduce

Check out the TiDB dashboard and view the slow query's detail information. The process time includes the suspended time. 
### What did you expect?
There should be standalone item called suspended time in TiDB
### What did happened?
The process time includes the suspended time.

### PR DESCRIPTION
**Pull request by glorv**
### What is changed and how it works?

Issue Number: Close #14262

What's Changed:

```
- Add TimeDetailV2 that return time related metrics in ns unit. 
- Add `process_suspend_wall_time_ns` in kv response, it's the schedule suspend wall time.
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

File: src/coprocessor/endpoint.rs [hunk 1]
```diff
@@ -1726,92 +1763,116 @@ mod tests {
             let resp = &rx.recv().unwrap()[0];
             assert!(resp.get_other_error().is_empty());
             assert_ge!(
-                resp.get_exec_details()
-                    .get_time_detail()
-                    .get_process_wall_time_ms(),
-                PAYLOAD_LARGE.saturating_sub(COARSE_ERROR_MS)
+                Duration::from_nanos(
```
Comment (nolouch) [inline src/coprocessor/endpoint.rs:1766]: Also add test for v1? make sure it's currently compatible.

Comment (glorv) [reply src/coprocessor/endpoint.rs:1774]: done. Add a check that the values in TimeDetail and TimeDetailV2 are equal.

**Comment by glorv** [comment]
@nolouch  PTAL again, thanks

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**